webpack.config.ts

```diff
@@ -11,6 +12,9 @@ const config = async (env: any): Promise<Configuration> => {
       // Required to load WASM modules.
       asyncWebAssembly: true,
     },
+    output: {
+      chunkFilename: 'ld.[chunkhash].js',
```

Comment:
Is this attempting to cache bust? We already do that with query strings in the default configs for prod.

https://github.com/grafana/logs-drilldown/blob/main/.config/webpack/webpack.config.ts#L182

Reply:
not attempting to cache bust but to allow users to cache assets across deploys when the contents of the chunks haven't changed